### PR TITLE
Update linear_search.c

### PR DESCRIPTION
**Pull request by RajaVamsi11**
Increased Documentation and Deleted unneccesary scrap....

### CODE REVIEW

File: search-algos/linear_search/l_search.c [hunk 1]
```diff
@@ -2,29 +2,31 @@
 #include<stdio.h>
 void main()
 {
-   int i,n,x,flag=0;
-   printf("enter how many value in array\n");  \\Taking no of elements in an array from user
-   scanf("%d",&n);
-   int a[n];
+   int flag,n,x;
+   printf("Enter the length of an Array\n");  
+   scanf("%d",&n);                                   \\Taking length of an array from user
```
Comment (22shubh22) [inline search-algos/linear_search/l_search.c]: comments start with // OR /* ....  */ .

File: search-algos/linear_search/l_search.c [hunk 2]
```diff
@@ -2,29 +2,31 @@
 #include<stdio.h>
 void main()
 {
-   int i,n,x,flag=0;
-   printf("enter how many value in array\n");  \\Taking no of elements in an array from user
-   scanf("%d",&n);
-   int a[n];
+   int flag,n,x;
```
Comment (22shubh22) [inline search-algos/linear_search/l_search.c]: Initialize the value of flag with 0, as you are using zero as the case when the element do not exit.

**Review by 22shubh22 — CHANGES_REQUESTED** [review]
change required in line 5,7 and all where comment is involved.

**Review by 22shubh22 — CHANGES_REQUESTED** [review]
Please reconsider your submission between line 23 to 30. Also use indentations in the code.

**Review by 22shubh22 — APPROVED** [review]

**Review by khanchi97 — APPROVED** [review]

**Pull request merged**